core/src/main/resources/error/error-classes.json

```diff
@@ -1278,6 +1288,11 @@
     ],
     "sqlState" : "42826"
   },
+  "OPERATION_NOT_ALLOWED" : {
+    "message" : [
+      "Operation not allowed: <message>."
```

Comment:
I think this one error class should have sub-classes instead of general `<message>`. Could you exclude it from the PR, please.

Reply:
done